src/content/app/help/components/help-menu/HelpMenu.test.tsx

```diff
@@ -45,7 +46,9 @@ describe('<HelpMenu>', () => {
 
       expect(
         topMenuItems.every((item) =>
-          renderedMenuItems.find((element) => element.textContent === item)
+          renderedMenuItems.find(
+            (element) => trim(element.textContent as string) === item
```

Comment:
Trim exists as a string method (`"foo   ".trim()` will give `'foo'`). I do wonder however if using a margin-left on the chevron wouldn't be preferable.